LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,6 +1,394 @@
-## The Slint Commercial License
+# **Slint Software License Agreement**
+Version 1.2, 22 August 2022
 
-Slint is offered under a commercial license. Please contact info@slint-ui.com if you're interested.
+Copyright © 2022 SixtyFPS GmbH <https://slint-ui.com/imprint.html>
 
-See also the [Licensing FAQ](https://github.com/slint-ui/slint/blob/master/FAQ.md#licensing) online
-or find the same information in `FAQ.md` in the top level of your slint repository checkout.
+
+SixtyFPS GmbH with offices at Am Panke-Park 47, 16321 Bernau
+bei Berlin, Germany (“**SixtyFPS**”) grants a license to the software 
```

Comment:
Do we need to use proper quotes here or would the ASCII quotes make do?

You are switch between "proper" quotes and ASCII quotes throughout this file, which I find very distracting.

Reply:
updated. now using only markdown syntax